### ISSUE TITLE
Start SVM does not work by just setting the admin_state to running

### ISSUE THREAD

**Issue by christian-naenny** [ansible-collections/netapp.ontap#76]
Following the implementation of Issue #37 I created a playbook to start/stop SVMs. The stop of the SVM works fine, but simply setting the parameter admin_state from stopped to running the playbook ends with an error.
Sample Code to stop the SVM:

```
- name: change the admin state of the SVM {{ svm }} to {{ admin_state }}
  netapp.ontap.na_ontap_svm:
    state: present
    name: {{ svm }}
    admin_state: stopped
    use_rest: always
    hostname: "{{ cluster_hostname }}"
    username: "{{ admin_username }}"
    password: "{{ admin_password }}"

```
With the following code I would expect the SVM to be started:
```
- name: change the admin state of the SVM {{ svm }} to {{ admin_state }}
  netapp.ontap.na_ontap_svm:
    state: present
    name: {{ svm }}
    admin_state: running
    use_rest: always
    hostname: "{{ cluster_hostname }}"
    username: "{{ admin_username }}"
    password: "{{ admin_password }}"

```
But unfortunately, this code throws an error:
```
Traceback (most recent call last):
  File \"/home/chn/.ansible/tmp/ansible-tmp-1654870265.8569968-519819-10884156135959/AnsiballZ_na_ontap_svm.py\", line 102, in <module>
    _ansiballz_main()
  File \"/home/chn/.ansible/tmp/ansible-tmp-1654870265.8569968-519819-10884156135959/AnsiballZ_na_ontap_svm.py\", line 94, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File \"/home/chn/.ansible/tmp/ansible-tmp-1654870265.8569968-519819-10884156135959/AnsiballZ_na_ontap_svm.py\", line 40, in invoke_module
    runpy.run_module(mod_name='ansible_collections.netapp.ontap.plugins.modules.na_ontap_svm', init_globals=None, run_name='__main__', alter_sys=True)
  File \"/usr/lib64/python3.6/runpy.py\", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File \"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File \"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code
    exec(code, run_globals)
  File \"/tmp/ansible_netapp.ontap.na_ontap_svm_payload_tugqwwy7/ansible_netapp.ontap.na_ontap_svm_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_svm.py\", line 816, in <module>
  File \"/tmp/ansible_netapp.ontap.na_ontap_svm_payload_tugqwwy7/ansible_netapp.ontap.na_ontap_svm_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_svm.py\", line 812, in main
  File \"/tmp/ansible_netapp.ontap.na_ontap_svm_payload_tugqwwy7/ansible_netapp.ontap.na_ontap_svm_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_svm.py\", line 758, in apply
  File \"/tmp/ansible_netapp.ontap.na_ontap_svm_payload_tugqwwy7/ansible_netapp.ontap.na_ontap_svm_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_svm.py\", line 508, in get_vserver
  File \"/tmp/ansible_netapp.ontap.na_ontap_svm_payload_tugqwwy7/ansible_netapp.ontap.na_ontap_svm_payload.zip/ansible_collections/netapp/ontap/plugins/modules/na_ontap_svm.py\", line 473, in clean_up_output
KeyError: 'cifs'
", "module_stdout": "", "msg": "MODULE FAILURE
See stdout/stderr for the exact error", "rc": 1}
```

**Comment by lonico** [comment]
which error?

**Comment by lonico** [comment]
Which ONTAP version?   It seems the CIFS value is not returned.  We did not see this in our tests.

**Comment by christian-naenny** [comment]
Right now we're on
NetApp Release 9.8P9: Wed Dec 22 09:39:04 UTC 2021

**Comment by lonico** [comment]
It seems you're using an older version of the module.  But I'm afraid the issue is also present in the latest version as well :(    
We'll work on a fix for this.

**Comment by lonico** [comment]
JIRA 5174

**Comment by lonico** [comment]
fixed in 21.21.0
